pipeline/cloud/schemas/cluster.py

```diff
@@ -4,3 +4,13 @@
 class PipelineClusterConfig(BaseModel):
     id: str
     node_pool: str
+
+
+class PipelineClusterGetLean(BaseModel):
+    """A lean representation of a cluster when returned from an API call"""
+
+    id: str
+    node_pool: str
```

Comment:
suppose you could inherit from `PipelineClusterConfig` to avoid copying these fields over?

Reply:
Yeah I did hesitate doing that. Went against it as `PipelineClusterConfig` is tied to `config.yaml` and it's not clear whether updating that in the future with some fields, whether those fields should also be part of the API response